types/webtorrent/index.d.ts

```diff
@@ -159,6 +159,8 @@ declare namespace WebTorrent {
         readonly length: number;
 
         readonly downloaded: number;
+        
```

Comment:
Looks like you might have some trailing whitespace on this line.

Reply:
Removed, thanks